packages/ivh-base-style/lib/css/table.css

```diff
@@ -129,3 +130,29 @@
   grid-column-start: start;
   grid-column-end: end;
 }
+
+.au-table-slim th,
+.au-table-slim td {
+  padding: 7px 10px;
+}
+
+.au-table-relaxed th,
+.au-table-relaxed td {
+  padding: 13px 10px;
+}
+
+
+.au-table-borderless {
+  border-top:none;
+}
+
+.au-table-borderless thead th,
+.au-table-borderless th,
+.au-table-borderless td,
+.au-table-borderless > * {
+  border: unset;
+}
+
+.au-table-striped tbody > tr:nth-child(odd){
+  background-color: #EFF0F1;
```

Comment:
You might consider adding this color to the theme list or using an already existing color if there's a good fit (https://github.com/iVantage/ivh-base/blob/master/packages/ivh-base-style/lib/index.js) and including it as a variable instead. If we end up adding the color, this has the added bonus of making that color available to use in code when importing the base styles package.